### PR TITLE
RS/JX/Rule16-7

### PR DESCRIPTION
**Pull request by Jiarongx-Xie**
This rule has been tested.

### CODE REVIEW

File: rct229/rulesets/ashrae9012019/section16/section16rule7.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            description="When included in the proposed design, the baseline elevator cab lighting power density shall be 3.14 W/ft2.",
+            ruleset_section_title="Elevators",
+            standard_section="Table G3.1-16 Baseline Building Performance",
+            is_primary_rule=True,
+            list_path="ruleset_model_descriptions[0].buildings[*].elevators[*]",
+        )
+
+    def is_applicable(self, context, data=None):
+        rmd_p = context.PROPOSED
+        return find_all("$.ruleset_model_descriptions[0].buildings[*].elevators", rmd_p)
```
Comment (yunjoonjung-PNNL) [inline rct229/rulesets/ashrae9012019/section16/section16rule7.py:34]: Do we need [*] at the end of the json path?

**Review by yunjoonjung-PNNL — APPROVED** [review]
Looks great. Thank you Jiarong.

File: rct229/rulesets/ashrae9012019/section16/section16rule7.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
+from rct229.schema.config import ureg
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.ruleset_model_factory import produce_ruleset_model_instance
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/section16/section16rule7.py:4]: update the import

File: rct229/rulesets/ashrae9012019/section16/section16rule7.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+from rct229.rulesets.ashrae9012019 import BASELINE_0, PROPOSED
```
Comment (weilixu) [inline rct229/rulesets/ashrae9012019/section16/section16rule7.py:5]: No need to import PROPOSED.

**Review by weilixu — COMMENTED** [review]
Similar issue as the other section 16 rules.

File: rct229/rulesets/ashrae9012019/section16/section16rule7.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            description="When included in the proposed design, the baseline elevator cab lighting power density shall be 3.14 W/ft2.",
+            ruleset_section_title="Elevators",
+            standard_section="Table G3.1-16 Baseline Building Performance",
+            is_primary_rule=True,
+            list_path="ruleset_model_descriptions[0].buildings[*].elevators[*]",
+        )
+
+    def is_applicable(self, context, data=None):
+        rmd_p = context.PROPOSED
+        return find_all(
```
Comment (yunjoonjung-PNNL) [inline rct229/rulesets/ashrae9012019/section16/section16rule7.py:34]: Could you follow https://github.com/pnnl/ruleset-checking-tool/pull/1342#:~:text=%40JacksonJ%2DKC%20%2C%20could%20you%20check%20what%20outcome%20should%20it%20be%20if%20the%20baseline%20has%20no%20elevator%20but%20proposed%20has%3F%0AThe%20rule%20passes%20if%20no%20elevator%20in%20baseline%20model%20but%20proposed%20has. and fix this is applicable logic?

**Pull request merged**